src/translate.rs

```diff
@@ -1855,6 +1937,120 @@ impl FromGlibPtrContainer<*const c_char, *mut glib_sys::GHashTable> for HashMap<
     }
 }
 
+impl<T> FromGlibContainerAsVec<<T as GlibPtrDefault>::GlibType, *mut glib_sys::GPtrArray> for T
+where
+    T: GlibPtrDefault
+        + FromGlibPtrNone<<T as GlibPtrDefault>::GlibType>
+        + FromGlibPtrFull<<T as GlibPtrDefault>::GlibType>,
```

Comment:
You duplicated this trait bound

Reply:
Na, just enforced it even more!

Reply:
Right, I copied from the same trait implementations for `SList` and `List`.

Reply:
Oh, can you also fix those then? Thanks :)